lib/Test/Nginx/Util.pm

```diff
@@ -2398,13 +2398,13 @@ END {
 sub can_run {
     my ($cmd) = @_;
 
-    #warn "can run: @_\n";
-    #my $_cmd = $cmd;
-    #return $_cmd if (-x $_cmd or $_cmd = MM->maybe_command($_cmd));
+    #warn "can run: $cmd\n";
+    return $cmd if -f $cmd && -x $cmd;
```

Comment:
I think we should check if `$cmd` contains slashes or back-slashes. Only when it looks more like a sole program name, should we do the `PATH` search.